qa/SidechainTestFramework/account/httpCalls/wallet/balance.py

```diff
@@ -1,7 +1,7 @@
 import json
 
 #executes a wallet/balance call on an account sidechain
-def http_wallet_balance(sidechainNode, evm_address, api_key = None):
+def http_wallet_balance(sidechainNode, evm_address, api_key = 'Horizen'):
```

Comment:
I'm not sure that default is `Horizen`, should be `None` as for all other methods.
Pls, check the usage.